### PR TITLE
set up power_seq_fsm files

### PR DESCRIPTION
**Pull request by harsh-patell**

### CODE REVIEW

**Comment by harsh-patell** [comment]
Temporary PR

File: projects/power_distribution/config.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
     "libs": [
         "FreeRTOS",
         "ms-common"
-    ]
+    ],
+    "can": true
```
Comment (JarvisWeng) [inline projects/power_distribution/config.json:6]: Did you not need to edit the `SConstruct`?

Comment (harsh-patell) [reply projects/power_distribution/config.json:6]: I have pushed the changes to SConstrust, which are temporary, and you mentioned that you would add some changes yourself. Do you want me to work on this or will you?

File: scons/build.scons [hunk 2]
```diff
@@ -136,12 +136,15 @@ for entry in PROJ_DIRS + LIB_DIRS + SMOKE_DIRS:
         #             - new_can
         # The CAN output should actually go into the projects/<project>
         # Fine for now but will be an issue if ever trying to build multiple projects
+        can_location_name = entry.name
+        if entry.name == "power_distribution":
+            can_location_name = "power_distribution_front"
```
Comment (JarvisWeng) [inline scons/build.scons:141]: Lol ok just change this part to be 
```
can_location_name += GetOption("location")
```
Maybe I'll do something better later on.... Probably not though

Comment (harsh-patell) [reply scons/build.scons]: I've added that

**Review by mitchellostler — APPROVED** [review]

**Pull request merged**